analyzer/habitat/analysis/predictor.py

```diff
@@ -140,7 +140,7 @@ def _wave_scale(self, run_time, dest_device):
         )
 
     def _special_scale(self, operation, dest_device, scaler):
-        predicted_ms = scaler(operation, dest_device)
+        predicted_ms, measured_local, predicted_local, unscaled_predicted = scaler(operation, dest_device)
```

Comment:
Can probably remove `measured_local` and `predicted_local`. 

We don't always need `unscaled_predicted`, so maybe include an optional parameter to `scaler` (e.g. `include_unscaled`) which enables returning this additional value, or just call it `scale=False` and call `scaler` twice. 

```python
predicted_ms = scaler(operation, dest_device) # existing API
unscaled_predicted = scaler(operation, dest_device, scale=False) # new additional option
```

Reply:
added optional parameter